src/kOS/Function/Misc.cs

```diff
@@ -373,4 +377,41 @@ public override void Execute(SharedObjects shared)
             ReturnValue = shared.Cpu.DumpVariables();
         }
     }
+
+    [Function("warpto")]
+    public class WarpTo : FunctionBase
+    {
+        public override void Execute(SharedObjects shared)
+        {
+            // TODO: As of KSP v1.0.2, the maxTimeWarping and minTimeWarping parameters behave unpredictably.  Disabling for now, we should revisit it in a later version.
+            int args = CountRemainingArgs(shared);
+            //double maxWarp = 8.0;
+            //double minWarp = 2.5;
+            double ut = 0.0;
+            switch (args)
+            {
+                //case 3:
+                //    minWarp = GetDouble(PopValueAssert(shared));
+                //    maxWarp = GetDouble(PopValueAssert(shared));
+                //    ut = GetDouble(PopValueAssert(shared));
+                //    break;
+
+                //case 2:
+                //    maxWarp = GetDouble(PopValueAssert(shared));
+                //    ut = GetDouble(PopValueAssert(shared));
+                //    break;
+
+                case 1:
+                    ut = GetDouble(PopValueAssert(shared));
+                    break;
+
+                default:
+                    throw new KOSArgumentMismatchException(new[] { 1 }, args);
+                    //throw new KOSArgumentMismatchException(new[] { 1, 2, 3 }, args);
+            }
+            AssertArgBottomAndConsume(shared);
+            TimeWarp.fetch.WarpTo(ut);
+            //TimeWarp.fetch.WarpTo(ut, maxTimeWarping: maxWarp, minTimeWarping: minWarp);
```

Comment:
@hvacengi i am going to remove the comments in this code because i dont think that they are accurate anymore?